src/containers/WithDraw.js

```diff
@@ -0,0 +1,127 @@
+import React, { Component } from "react";
+import Web3 from 'web3';
+import FestakingJson from "../contracts/Festaking.json";
+import FerrumJson from "../contracts/FerrumToken.json"
+import WithdrawForm from '../components/WithdrawForm';
+
+class WithDraw extends Component {
+    state = {
+        web3: null,
+        contract: null,
+        amount: 0,
+        festaking:null,
+        GAS: 600000,
+        toWithdraw:'',
+        withdrawTime:'',
+        owner:null,
+        STAKING_CAP: 1000,
+        frm: null,
+    };
+
+    async componentDidMount(){
+        const web3 = new Web3("ws://localhost:7545");
+        const accounts = await web3.eth.getAccounts();
+        const owner = accounts[0];
```

Comment:
We only care about one user account. For testing please initialize the accounts in different scripts